app/Locations.tsx

```diff
@@ -2,15 +2,38 @@ import { ReactElement, useState } from "react";
 import { PageHeader } from "./ui/PageHeader";
 import { Wrapper } from "./ui/Wrapper";
 import { ViewLocations } from "./ViewLocations";
+import { CreateLocations } from "./CreateLocations";
+import { LargeButton } from "./ui/LargeButton";
+import { Button } from "react-bootstrap";
+import { greenActiveButton, greyInactiveButton } from "./ui/colourLibrary";
 
 enum LocationPage {
   View,
+  Create,
 }
 
 export function Locations(): ReactElement {
   const [locationPage, setLocationPage] = useState(LocationPage.View);
 
   return (
-    <Wrapper>{locationPage === LocationPage.View && <ViewLocations />}</Wrapper>
+    <Wrapper>
+      {locationPage === LocationPage.View && (
+        <ViewLocations
+          onCreateClick={() => {
+            setLocationPage(LocationPage.Create);
+          }}
+        />
+      )}
+      {locationPage === LocationPage.Create && (
+        <CreateLocations
+          onCancelClick={() => {
+            setLocationPage(LocationPage.View);
+          }}
+          onCreateLocationClick={() => {
+            setLocationPage(LocationPage.View);
+          }}
```

Comment:
this would probably be `onCreateLocationSuccess`, if the submission doesn't work you probably wouldn't want to go back to `View`. Can change it later when you actually implement the creation